local-interchaintest/examples/ibc_transfer_juno_ntrn.rs

```diff
@@ -0,0 +1,240 @@
+use std::{
+    env,
+    error::Error,
+    ops::{Add, Sub},
+};
+
+use cosmwasm_std_old::Coin as BankCoin;
+use local_interchaintest::utils::{
+    base_account::{approve_service, create_base_accounts},
+    GAS_FLAGS, LOGS_FILE_PATH, NTRN_DENOM, VALENCE_ARTIFACTS_PATH,
+};
+use localic_std::modules::{
+    bank,
+    cosmwasm::{contract_execute, contract_instantiate},
+};
+use localic_utils::{
+    ConfigChainBuilder, TestContextBuilder, DEFAULT_KEY, GAIA_CHAIN_NAME, JUNO_CHAIN_ADMIN_ADDR,
+    JUNO_CHAIN_DENOM, JUNO_CHAIN_NAME, LOCAL_IC_API_URL, NEUTRON_CHAIN_NAME,
+};
+use log::info;
+
+use valence_ibc_transfer_service::msg::{ActionsMsgs, ServiceConfig};
+use valence_service_utils::{denoms::UncheckedDenom, ServiceAccountType};
+
+fn main() -> Result<(), Box<dyn Error>> {
+    env_logger::init();
+
+    let ibc_denom = "ibc/4E41ED8F3DCAEA15F4D6ADC6EDD7C04A676160735C9710B904B7BF53525B56D6";
```

Comment:
this won't always work, channels are not consistent in local-ic so you might get different channels sometimes.
We have helpers to get the ibc denom after we have built the test_ctx. Examples:

```rust
let neutron_denom = test_ctx.get_native_denom().src(NEUTRON_CHAIN_NAME).get();
let neutron_on_juno_denom =  test_ctx
        .get_ibc_denom()
        .base_denom(neutron_denom)
        .src(NEUTRON_CHAIN_NAME)
        .dest(JUNO_CHAIN_NAME)
        .get();
```